tests/__init__.py

```diff
@@ -7,14 +7,42 @@
 
 from sqlalchemy_defaults import make_lazy_configured
 
+IS_MYSQL = (
+    os.environ.get('DB') == 'mysql' or
+    os.environ.get('DSN', '').startswith('mysql')
+)
+MYSQLD_VERSION = tuple(
```

Comment:
I think there is some better SQLAlchemy way of getting the driver version (I can't remember if it was from connection or dialect object that returned this).